### ISSUE TITLE
关于预训练数据集objects365v2

### ISSUE THREAD

**Issue by CrapbagMo** [YifanXu74/MQ-Det#48]
您好，您在 #44 中提到预训练数据集是objects365v2。我正在尝试复现您的结果，请问在modulated pretraining 阶段您是从objects365v2中采样了0.66M的数据吗，还是使用了整个数据集。

**Comment by CrapbagMo** [comment]
您好，我还有一个问题。我发现GLIP使用的objects365v1数据集，各个train标注文件中显示却只有0.608M图像，请问您知道GLIP中的0.66M这个数据是怎么产生的吗?
[train.label.lineidx.txt](https://github.com/YifanXu74/MQ-Det/files/14821424/train.label.lineidx.txt)

**Comment by dbcSep03** [comment]
objv1就是0.60的数据集 0.66M出自GLIP但是似乎不对，在Grounding Dino中有“There are two versions of the O365 dataset, which we termed O365v1 and O365v2, respectively. O365v1 is a subset of O365v2. O365v1 contains about 600K images, while O365v2 contains about 1.7M images. Following previous works [26, 53], we pre-train the Grounding-DINO-T on O365v1 for a fair comparison. The Grounding-DINO-L is pre-trained on O365v2 for a better result.”
